api/cache/projects.js

```diff
@@ -174,25 +174,30 @@ async function getInitialTriageProjectsFromJira(startAt, itArray) {
 
   const itTotalData = parseInt(jiraIt.data.total);
 
-  jiraIt.data.issues.map((x) =>
-    itArray.push({
+  await Promise.all(jiraIt.data.issues.map(async (x) => {
+    const project = {
       it_key: x['key'],
       name: x['fields'].summary,
       description: x['fields'].description,
       client: x['fields'].customfield_10027,
       video: x['fields'].customfield_10159 ?? '',
       scope: x['fields'].customfield_10090,
-      sector: x['fields'].customfield_10148?.value ?? ''
-    }),
-  );
+      sector: x['fields'].customfield_10148?.value ?? '',
+    }
+
+    const videoFile =  await videoHelper.getVideoFileFromVimeo(project.video);
+    project.video_file = videoFile;
+
+    itArray.push(project);
+  }));
```

Comment:
Hey Rosie, thanks for the review! I tried your suggestion but I'm getting an error when running the cache job - not sure why as everything looks to be running in the same order as what my code is doing it currently. Did it work for you? Might be I've done something wrong when implementing it

Reply:
Hiya @CatMac228, just pulled your code and had a look. Not sure which error you're getting, but there are a couple that I can see it might be, not to do with this change:
1.  The script seems to be sensitive to where it's run from: running `node cache/run-projects.js` from the `api` directory works for me, whereas `node run-projects.js` from the `api/cache` directory doesn't. I haven't looked into why that's the case, but if it throws a big long error message, check where you're running it from
2. When I run `node cache/run-projects.js` it says "❌ Could not save new projects/resources records in cache". If you add a `console.error(error)` in the catch block where this error message is being printed (in the `cacheProjectsAndResources` function), then you get this:
![Capture](https://user-images.githubusercontent.com/13078984/183026446-cf2e19cc-9da6-4a00-9102-05c5e77efc57.PNG)
Which suggests that it's failing because Airtable doesn't like the renamed video/video_webpage field. Are we able to rename the field in Airtable? If not, then another option would be to adjust the code to put the field name back to how it was, even if only for when it's sent to Airtable.

If you're getting a different error to do with this block of code, shout and I'll see what I can do :-)